src/components/BounceRulesContainer/index.test.jsx

```diff
@@ -28,7 +28,30 @@ it("should render correctly", () => {
   const tree = renderer
     .create(
       <BounceRuleContainer
-        filteredRules={[]}
+        rules={[
+          {
```

Comment:
since it looks like the contents in `rules` and `filteredRules` are the same, you can store it in a variable for reusability
i.e. store the reused code in something like `testRules` so you can do
```
rules={testRules}
filteredRules={testRules}